### ISSUE TITLE
Method arguments in parameter tables don't all render correctly

### ISSUE THREAD

**Issue by ixjlyons** [mdolab/pyXDSM#39]
In the [hosted API docs](https://mdolab-pyxdsm.readthedocs-hosted.com/API.html), some of the method arguments have no delimiting " : " between the name and type. See below:

![image](https://user-images.githubusercontent.com/943602/113202999-51910780-9220-11eb-98ce-ec50952d5556.png)

The ones that do look correct have no space between the name and the type(s) in the docstring.

When I build the docs locally, this issue doesn't occur:

![image](https://user-images.githubusercontent.com/943602/113203208-97e66680-9220-11eb-9772-3ca3ffcfeb0c.png)

I also notice the table overall looks a bit different. Perhaps the sphinx theme or numpydoc version needs to be updated on the server hosting the docs.

**Comment by nwu63** [comment]
Yeah we noticed and fixed this recently -- funnily enough, the RTD theme is outdated on RTD. I just manually rebuilt the docs and they should look fine now (you may need to F5 a few times).

**Comment by ixjlyons** [comment]
Looks good now, thanks!
